languages/csharp/code/.codecrafters/run.sh

```
@@ -8,4 +8,4 @@
 
 set -e # Exit on failure
 
-exec /tmp/codecrafters-build-csharp/codecrafters-{{course_slug_underscorized}} "$@"
+exec /tmp/codecrafters-build-csharp/CodeCrafters.{{course_slug_underscorized}} "$@"
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)